tests/xharness/AppRunner.cs

```diff
@@ -334,6 +334,23 @@ public async Task<ProcessExecutionResult> UninstallAsync ()
 			}
 		}
 
+		// test if the file is valid xml, or at least, that can be read. The reader will throw an 
+		// exception if it cannot read the file. This is not ideal :/
+		bool IsValidXml (string filePath)
+		{
+			if (!File.Exists (filePath))
+				return false;
+
+			try {
+				using (var stream = new StreamReader (filePath)) 
+				using (var reader = XmlReader.Create (stream)) {
```

Comment:
double check that does not use/create an XmlResolver that will try to do network access